src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java

```diff
@@ -39,7 +40,7 @@
 
     public void ingestPaymentOrders(String externalUserId) {
         final List<String> PAYMENT_TYPES = Arrays
```

Comment:
not releated to your PR but you should either define this as a constant (then uppercase is correct) or just camelcase it (or rename to `ootbPaymentTypes`

Reply:
done